### DISCUSSION TITLE
Não consigo conectar 2 controles bluetooth ao mesmo tempo, para poder jogar duas pessoas.

**I can't connect 2 bluetooth controllers at the same time, so I can play two people.**

### DISCUSSION THREAD

**Discussion by Alefox007** [darthcloud/BlueRetro#561] [Q&A] [answered]
Não consigo conectar 2 controles bluetooth ao mesmo tempo, os dois controles são da marca "ipega" e de modelos diferentes.O primeiro é "Ipega 9025"' mais antigo, e o outro é o "Ipega 9076", os dois controles funcionam perfeitamente porém só consigo conectar apenas um deles. Resumindo: Os dois controles conectam perfeitamente, mas não consigo parear os juntos para jogar jogos multiplayers.

I can't connect 2 bluetooth controllers at the same time, the two controllers are from the "ipega" brand and different models. The first is the older "Ipega 9025", the other is the "Ipega 9076", both controllers work perfectly but I can only connect only one of them. Bottom line: The two controllers connect perfectly, but I can't pair them together to play multiplayer games.

How to pair two ipega controllers on BlueRetro at the same time?

**Comment by hardlevel** [comment]
você so pode conectar dois ou mais em blueretro que tem duas ou mais conexões de cabo, aquele simples do aliexpress pra ps1 e ps2 que serve so em um slot não da pra conectar dois

**Comment by Alefox007** [comment]
> você so pode conectar dois ou mais em blueretro que tem duas ou mais conexões de cabo, aquele simples do aliexpress pra ps1 e ps2 que serve so em um slot não da pra conectar dois

O meu é aquele núcleo que vem com o cabo a sua escolha, no meu caso escolhi de PS2 que no caso conecta nos dois slots, o jogo até reconhece que tem dois controles na porta porém não consigo parear os dois simultaneamente... Talvez pelo fato de ser da marca ipega e de modelos diferentes.

**Comment by hardlevel** [comment]
veja nas configurações na interface web la conectando pelo celular, se a
opção do multitap tá ativa, tem que estar em none, o multitap não funciona
no ps2

Em dom., 15 de jan. de 2023 às 23:03, Alefox007 ***@***.***>
escreveu:

**Comment by Alefox007** [comment]
> veja nas configurações na interface web la conectando pelo celular, se a
> opção do multitap tá ativa, tem que estar em none, o multitap não funciona
> no ps2


**Já está a assim, sem sucesso ainda...**


![Screenshot_2023-01-16-09-58-46-488_com.android.chrome.jpg](https://user-images.githubusercontent.com/122080657/212684456-6266d9fb-e2ca-4684-8bfd-a8bfe80c12cd.jpg)

**Comment by Alefox007** [comment]
Já tentei colocar na configuração global o PS2 mais não vai também

**Comment by hardlevel** [comment]
Vc já tentou configurar os dois outputs nessa tela aí da imagem? Configura
o 1 e o 2

Em seg., 16 de jan. de 2023 10:08, Alefox007 ***@***.***>
escreveu:

**Comment by Alefox007** [comment]
Hmmm, vou testar pra ver...

**Comment by Alefox007** [comment]
É também não adiantou

**Comment by Alefox007** [comment, marked as answer]
**RESOLVIDO**

1° - Ligue o BlueRetro
2° - Ligue o 1° controle e pareie
3° - Ligue o 2° controle
4° - Segure o botão boot da luz azul por 2 ou 3 segundos e solte quando ele piscar e aguarde ele se conectar com o 2 controle!


**SORTED OUT**

1° - Turn on BlueRetro.
2° - Turn on the 1st controller and pair it.
3° - Turn on the 2nd controller.
4° - Hold the blue light boot button for 2 or 3 seconds and release when it blinks and wait for it to connect with the 2nd controller!

**Reply by joaomarcos70** [reply]
boa noite!

estou tentando fazer esse mesmo metodo para conectar dois controles aqui no meu ps2 mas não estou conseguindo, eu comprei dois adaptadores blue retro e mesmo assim não consigo, consegue me ajudar @Alefox007 ?
